motoman_msgs/srv/WriteGroupIO.srv

```diff
@@ -0,0 +1,25 @@
+
+# Write 'value' to the Group IO element at 'address'.
+#
+# This service does not return anything.
```

Comment:
Technically it returns a boolean and string

Reply:
Yes, true. I already addressed this for #439 in 8d5a823af05860d40a819242d36318c76ae045ab.

I'll add a similar commit here.

Reply:
Fixed: 81f7851e.